ddsketch/encoding/encoding.go

```diff
@@ -162,3 +188,21 @@ func DecodeVarfloat64(b *[]byte) (float64, error) {
 	*b = (*b)[i+1:]
 	return math.Float64frombits(bits.RotateLeft64(x, -varfloat64Rotate)+math.Float64bits(1)) - 1, nil
 }
+
+// Varfloat64Size returns the number of bytes that EncodeVarfloat64 encodes a
+// 64-bit floating-point value into.
+func Varfloat64Size(v float64) int {
```

Comment:
im just assuming you got this right because I don't understand it

Reply:
There's some explanation above (see `EncodeVarfloat64`).